crates/turborepo-ui/src/tui/event.rs

```diff
@@ -64,6 +68,7 @@ pub enum Event {
     SearchBackspace,
 }
 
+#[derive(Clone)]
```

Comment:
Total nit, but for this we should do a `Copy` implementation as well

```suggestion
#[derive(Copy, Clone)]
```